### PR TITLE
Modify features section to explain instance of multiple inputs with same prefix

### LINKED ISSUE #127: [PE-D] Multiple instances of the same prefix are not handled
Specifically for adding a new flash card, If i were to use 2 of the same prefixes such as `a/ANSWER1` and `a/ANSWER2`, ANSWER2 would be reflected.

Steps to replicate:
1. type add q/Question a/answer1 a/answer2

similar case for other commands and other duplicate prefixes.

Suggestions:
Throw an error message indicating that you should only specify 1 prefix.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: davidliew9/ped#3

### LINKED ISSUE #135: [PE-D] Two note fields can be present and only one shows up
maybe can combine two note fields, if there are any

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: juandavinlie/ped#2

### LINKED ISSUE #138: [PE-D] Multiple category parameters work and Swee picks the latest one
Users may copy and paste some input and forgot some fields that they have already specified. This may lead to unintended result.

-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: juandavinlie/ped#5

### LINKED ISSUE #147: [PE-D] Filtering same multiple fields work, but only shows the latest
-------------
Labels: `severity.Medium` `type.FunctionalityBug`
original: juandavinlie/ped#14

### PR DESCRIPTION
**Pull request by tan-zhuoyao**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -84,8 +85,17 @@ SWEe! is a  **desktop app for CS2103T students to manage their learning progress
 * Words in lower_case are to be specified exactly. e.g. in `sort <success|reviewed>`, `success` and `reviewed` 
 must be specified exactly.
 
-* Items in square brackets are optional.<br>
-  e.g `q/QUESTION [c/CATEGORY]` can be used as `q/What is my name? c/topic 1` or as `q/What is my name?`
+* Items in square brackets are optional (zero or one instance).<br>
+  e.g `q/QUESTION [c/CATEGORY]` can be used as `q/What is my name?` or as `q/What is my name? c/topic 1`
+  * Note: Multiple instances of the same prefix will not throw an error. 
+  However, the software will only read the last instance in the input.
```
Comment (sgn00) [inline docs/UserGuide.md]: "app" instead of "software"

File: docs/UserGuide.md [hunk 2]
```diff
@@ -84,8 +85,17 @@ SWEe! is a  **desktop app for CS2103T students to manage their learning progress
...
+* Items in square brackets are optional (zero or one instance).<br>
+  e.g `q/QUESTION [c/CATEGORY]` can be used as `q/What is my name?` or as `q/What is my name? c/topic 1`
+  * Note: Multiple instances of the same prefix will not throw an error. 
+  However, the software will only read the last instance in the input. 
+  * e.g `add q/Are apples red? a/Yes c/fruit c/test` will add a flashcard belonging to "test" category 
+  
+* Items in square brackets with trailing ellipsis are optional (zero, one or multiple instances).<br>
+  e.g. `q/QUESTION [t/TAG]...` can be used as `q/What is my name?`, `q/What is my name? t/topic1` or
+  `q/What is my name? t/topic1 t/fun` 
+  * Note: Multiple instances of the tag prefix is accepted, and the software will read every instance
```
Comment (sgn00) [inline docs/UserGuide.md]: I think we can use "app" instead of "software"

**Review by sgn00 — CHANGES_REQUESTED** [review]
Refer to comments.

File: docs/UserGuide.md (hunk 2 above)
Comment (tan-zhuoyao) [reply docs/UserGuide.md]: Got it!

File: docs/UserGuide.md (hunk 1 above)
Comment (tan-zhuoyao) [reply docs/UserGuide.md]: Got it!

**Pull request merged**